mongo/description/selector_test.go

```diff
@@ -231,32 +231,69 @@ func TestSelector_Sharded(t *testing.T) {
 }
 
 func BenchmarkSelector_Sharded(b *testing.B) {
-	subject := readpref.Primary()
-
-	s := Server{
-		Addr:              address.Address("localhost:27017"),
-		HeartbeatInterval: time.Duration(10) * time.Second,
-		LastWriteTime:     time.Date(2017, 2, 11, 14, 0, 0, 0, time.UTC),
-		LastUpdateTime:    time.Date(2017, 2, 11, 14, 0, 2, 0, time.UTC),
-		Kind:              Mongos,
-		WireVersion:       &VersionRange{Min: 0, Max: 5},
-	}
-	servers := make([]Server, 100)
-	for i := 0; i < len(servers); i++ {
-		servers[i] = s
-	}
-	servers[0].Kind = LoadBalancer
-	c := Topology{
-		Kind:    Sharded,
-		Servers: servers,
+	for _, bcase := range []struct {
+		name        string
+		serversHook func(servers []Server)
+	}{
+		{
+			"AllFit",
+			func(servers []Server) {},
```

Comment:
[Optional] It will be better if we can add field names like: 
```suggestion
			name:        "AllFit",
			serversHook: func(servers []Server) {},
```

Reply:
Added in all 4 places